README.md

```diff
@@ -97,42 +97,41 @@ brew install gopinath-langote/one-build/one-build
   ```
 
 ### Using `before` and `after` commands
-Consider that your project requires some environment variables to set before running any 
+Consider that your project requires some environment variables to set before running any
 commands and you want to clean up those after running commands. It is a headache to always
-remember to set those environment variables. What you want is to set env variables automatically 
-when you run the command in the project and remove those when the command is complete. 
+remember to set those environment variables. What you want is to set env variables automatically
+when you run the command in the project and remove those when the command is complete.
 Another example – a project requires `Docker` to be up
 and running or you need to clean up the database after running a test harness.
 
 This is where `before` & `after` commands are useful. These commands are both optional – 
 you can use one of them, both or neither.
 
-#### Examples
-1.  Setting env variables and cleaning those up
-    ```yaml
-    project: Sample Web App
-    before: export VARNAME="my value"
-    after: unset VARNAME
-    commands:
-      - build: npm run build
+#### Examples:
+1. Setting env variables and cleaning those up
+    ```console
+    1build set before 'export VARNAME="my value"'
+    1build set after "unset VARNAME"
     ```
 
-2.  Ensure that `Docker` is up and running
-    ```yaml
-    project: Containerized Project
-    before: ./docker_run.sh
-    commands:
-      - build: npm run build
+2. Ensure that `Docker` is up and running
+    ```console
+    1build set before "./docker_run.sh"
     ```
 
-3.  Clean up database after some commands
-     ```yaml
-    project: Containerized Project
-    after: ./clean_database.sh
-    commands:
-      - build: npm run build
+3. Clean up database after some commands
+     ```console
+    1build set after "./clean_database.sh"
     ```
 
+### Removing `before` and `after` commands
```

Comment:
Let's not make a separate section for ```Removing `before` and `after` commands```

The reason behind creating a section for before and after was - to convinence the idea behind it.

Let's have one section of Before and after – and just examples for setting and unsetting it?

WDYT?

Reply:
sure. sounds reasonable to me. gonna change it now